dcat/index.html

```diff
@@ -2866,6 +2869,48 @@ <h4>Property: packaging format</h4>
             <p>For examples on the use of this property, see <a href="#examples-compressed-and-packaged-distributions"></a>.</p>
 
         </section>
+		
+
+<section id="Property:distribution_checksum">
+
+<h4>Property: checksum</h4>
+
+<aside class="note">
+<p>Property added in DCAT 3.</p>
+</aside>
+
+<table class="definition">
+<thead>
+<tr>
+<th>RDF Property:</th>
+<th><a data-cite="SPDX#d4e1930" title="http://spdx.org/rdf/terms#checksum">spdx:checksum</a></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td class="prop">Definition:</td>
+<td>A Checksum is value that allows the contents of a file to be authenticated. Even small changes to the content of the file will change its checksum. This class allows the results of a variety of checksum and cryptographic message digest algorithms to be represented [[SPDX]].</td>
```

Comment:
I see that you took this definition from SPDX, but I think it is worth to add some fixes/changes to the text and say that the definition is based on SPDX definition, rather than taking it verbatim.

I suggest:
- change 'A Checksum is value' to 'A checksum is a value'
- change 'that allows the contents of a file to be authenticated' to 'that allows to check the integrity of the contents of a file'

Reply:
Also - this definition should be about the property. We need to include the class definition elsewhere.

Reply:
Actually - the class definition is again below.

Reply:
Many thanks for spotting this inconsistency, @agbeltran . 

I fixed the definition of `sdpx:checksum`, and the typo in the definition of `spdx:Checksum` ("A Checksum is a value").

About your suggestion of revising the SPDX definitions, I added a usage note with text suggested by @riannella in https://github.com/w3c/dxwg/issues/1287#issuecomment-798901465 along your lines (I think it is important we keep the original definition and add information on how its use in DCAT separately).